include/boost/beast/http/fields.hpp

```diff
@@ -478,6 +491,12 @@ class basic_fields
         comparison, otherwise the behavior is undefined.
 
         @param value The value of the field, as a @ref boost::beast::string_view
+
+        @throws boost::system::system_error Thrown if an error occurs:
+        - If the size of @c name_string exceeds @ref max_name_size, the
```

Comment:
should this use @li or is there a docca bug preventing it?

Reply:
I replaced it with @li and it works fine.